spec/classes/jenkins_service_spec.rb

```diff
@@ -76,6 +77,27 @@
           end
         end
       end
+      if os_facts[:os]['release']['major'].to_i >= 7
```

Comment:
I'd prefer to rebase this on https://github.com/voxpupuli/puppet-jenkins/pull/1026. That removes the need for this conditional.

Reply:
This now merged. Could you rebase?

Reply:
I thinks it's ok like that ?